src/hotspot/os_cpu/linux_riscv/vm_version_linux_riscv.cpp

```diff
@@ -102,6 +120,27 @@ void VM_Version::setup_cpu_available_features() {
   if (!RiscvHwprobe::probe_features()) {
     os_aux_features();
   }
+
+  // Linux kernel require Zifencei
+  if (!ext_Zifencei.enabled()) {
+    ext_Zifencei.enable_feature();
```

Comment:
That would deserve a `log_info` of sort, to make sure we can debug such state change in the wild